README.md

```diff
@@ -1,25 +1,27 @@
 # :closed_lock_with_key: BIP38
+
 This is a fork from [decentropy/bip38](https://github.com/decentropy/bip38) updated to **Python 3.10** and an additional support for other coins.
 
 This implementation supports
+
 * BTC (mainnet / testnet)
 * LTC (mainnet / testnet)
-* DOGE (mainnet / testnet) 
+* DOGE (mainnet / testnet)
 
-# :zap: Requirements
+## :zap: Requirements
 
 Required packages: (apt-get install) python-dev libssl-dev libjpeg-dev zlib1g-dev libpng-dev libfreetype6-dev
 
 Required modules: (pip install) pycrypto scrypt bitcoin base58 pillow qrcode cryptos
```

Comment:
Probably should mark `pycryptodome` here too.

Reply:
Yes, either that or just a reference to `requirements.txt`